### DISCUSSION TITLE
About Our Class Diagram

### DISCUSSION THREAD

**Discussion by MucahitErdoganUnlu** [bounswe/bounswe2024group11#176] [General]
I think our class diagram has multiple problems. To start with, consider creating a post:
* Our registered user class has no method called "create post"
* Create post only appears in "Contents" class, which manages (?) "Content" class.
In this setup, the user cannot create a post. Can someone clarify this?
@MucahitErdoganUnlu @umitcan07 @ceydanursen @Meminseeker @meminciftci @tenacke @hks1444 @ArdaVural @yunuskaydin @ozankrkya 
I found a helpful website for our social media class diagram: [link](https://github.com/tssovi/grokking-the-object-oriented-design-interview/blob/master/object-oriented-design-case-studies/design-facebook.md#class-diagram)

**Comment by Meminseeker** [comment]
Wow! What a wonderful repo. I'm impressed. Thanks @MucahitErdoganUnlu

**Comment by Meminseeker** [comment]
I guess @tenacke or @hks1444 can answer your question

**Comment by tenacke** [comment]
I guess the method createPost() can be added to the registered user class as a starting point for the create post sequence. What do you think @hks1444

**Reply by hks1444** [reply]
I agree
